content/en/blog/_posts/2024-03-19-go-workspaces.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: blog
+title: 'Using Go workspaces in Kubernetes'
+date: 2024-03-19T08:30:00-08:00
+slug: go-workspaces-in-kubernetes
+canonicalUrl: https://www.kubernetes.dev/blog/2024/03/19/go-workspaces-in-kubernetes/
+---
+
+**Author:** Tim Hockin (Google)
+
```

Comment:
We could add some text to explain that this is reposting the content published in March at https://www.k8s.dev/blog/2024/03/19/go-workspaces-in-kubernetes/

**I don't understand our overall intent here**; the improvement is great, but it's already delivered and end users won't notice it. We can have it as a post-release blog article but I don't understand why we would.

Reply:
This is to be published post-release because it was handled and tracked by the Release Comms team as a feature blog. It should not have been published without their knowledge or without them assigning a publication date. The original PR included all of that context. We reverted because it was, to the Release Team, an unauthorized early publication.

If it should never have been a feature blog and thus should never have been under the control of Release Comms, that's fine and it can be published whenever you and @thockin please, but as long as Release Comms owns it, Release Comms needs to be kept in the loop and approve of publication.

No objection to removing this from Release Comms ownership if it's regarding a feature that is already live and they don't have a problem with it, but everyone has to be informed of that.

Reply:
I truly truly do not care and am happy to roll with <whatever>.  It's not a "feature" of 1.30 - it's in the codebase now and any developer using k/k is subject to it.  That said, it was a KEP and tracked.

Reply:
For the less clueful - it's already published at https://www.k8s.dev/blog/2024/03/19/go-workspaces-in-kubernetes/ - so if we say this is NOT a feature blog, do I just close this PR and be done?  Any other implications?

Reply:
We have no objections, go ahead and close it.